checkov/sast/engines/prisma_engine.py

```diff
@@ -311,16 +312,15 @@ def create_report(self, prisma_report: PrismaReport) -> List[SastReport]:
                     file_abs_path = location.path
                     file_path = file_abs_path.split('/')[-1]
                     file_line_range = [location.start.row, location.end.row]
-
+                    split_code_block = [line + '\n' for line in location.code_block.split('\n')]
+                    code_block = get_code_block_from_start(split_code_block, location.start.row)
+                    metadata = None
                     if match.metadata.taint_mode is not None:
```

Comment:
```suggestion
                    metadata = _Metadata(taint_flow=get_data_flow_code_block(match.metadata.taint_mode.data_flow)) if match.metadata.taint_mode else None
```
I think this or extracting the population logic to another function and here just call `metadata = parse_metadata(match.metadata)` are better approaches, rather than declaring the `metadata = None` and then conditionally overriding it (with only one option). At least in python its not needed in this case